pyproject.toml

```diff
@@ -12,6 +12,7 @@ extend-exclude = '''
 '''
 
 [tool.isort]
+py_version=311
```

Comment:
We don't want to prescribe a specific python version right?

Reply:
I think we do, this way everyone's `isort` will format the same way regardless of their local Python interpreter version.

Reply:
Good point 👍